src/pages/gettingstarted/vanilla.hbs

```diff
@@ -0,0 +1,313 @@
+---
+title: Vanilla
```

Comment:
maybe we should change Vanilla to HTML, i dont think its a great name, but it matches the code switch

Reply:
Updated to html throughout